src/percy-agent-client/percy-agent.ts

```diff
@@ -101,8 +102,9 @@ export default class PercyAgent {
     return domClone
   }
 
-  private stabilizeDOM(dom: HTMLElement) {
-    const stabilizedDOM = this.serializeInputElements(dom)
+  private stabilizeDOM(doc: Document): HTMLElement {
+    const docWithSerializedCSSOM: Document = serializeCssOm(doc)
```

Comment:
I think about `stabilizeDOM()` in terms of a series of mutations on the input DOM. This call in particular feels a bit strange:

```typescript
this.serializeInputElements(docWithSerializedCSSOM.documentElement)
```

I'm wondering if something like this would be nicer:

```typescript
let stabilizedDOM = doc
stabilizedDOM = serializeCssOm(stabilizedDOM)
stabilizedDOM = this.serializeInputElements(stabilizedDOM)
```

And make a little update to `serializeInputElements` to take in a `HTMLDocument` not a document element.

Thoughts?

Reply:
Yeah, I agree that is nicer. Implemented and added to this PR.